win32ss/reactx/ntddraw/dvp.c

```diff
@@ -322,15 +311,14 @@ NtGdiDvpAcquireNotification(HANDLE hVideoPort,
 
     if (pfnDvpAcquireNotification == NULL)
     {
-		DPRINT1("Warning: no pfnDvpAcquireNotification");
+		DPRINT1("Warning: no pfnDvpAcquireNotification\n");
```

Comment:
tab indentation, and the neightbors don't have it. same as in the other cases. Should be fixable. And all is our code.